styles/index.css

```diff
@@ -98,47 +262,198 @@ body {
   text-align: left;
 }
 
-/* nav */
-#headline-nav {
-  margin: 0 0;
+/* Works */
+#works { padding: 120px 24px; }
```

Comment:
- [ ] As demanded by the project requirements, kindly use CSS grid to layout the contents of your work section. Grid helps in creating a responsive web page so it is important you practice using it.